ansible/roles/bootstrap/tasks/partials/fedora.yml

```diff
@@ -7,20 +7,6 @@
 #        use our own logic. Perhaps better to avoid ansible facts altogether?
 #
 
-- name: check for python
-  raw: stat /usr/bin/python2
```

Comment:
I assume we can drop Python 2 since Node.js 14 is EoL.

Reply:
Yes, we shouldn't need to install Python 2 on new hosts -- I think the only thing that might still need Python 2 is building/testing V8 on older Node.js release lines (e.g. 16) but we do not do that on the Fedora machines.